Machine-Learning/README.md

```diff
@@ -0,0 +1,106 @@
+- [Data analysis with Python -Tutorialspoint](https://www.tutorialspoint.com/data-analysis-and-visualization-in-python)
+- [Data analysis with Python -GeeksforGeeks](https://www.geeksforgeeks.org/data-analysis-visualization-python/)
+
+## Dive into Machine Learning
+- [Machine Learning -by Andrew Ng](https://youtube.com/playlist?list=PLoROMvodv4rMiGQp3WXShtMGgzqpfVfbU)
+- [Machine Learning -by Andrew Ng(All in One)](https://youtube.com/playlist?list=PLLssT5z_DsK-h9vYZkQkYNWcItqhlRJLN)
+- [Machine Learning full course for begineer - FreeCodeCamp](https://youtu.be/NWONeJKn6kc)
+- [Machine Learning with Python -Sentdex(Youtube)](https://youtube.com/playlist?list=PLQVvvaa0QuDfKTOs3Keq_kaG2P55YRn5v)
+- [Hands-On Machine Learning with Scikit-Learn, Keras, and TensorFlow by Aurélien Géron -pdf](http://powerunit-ju.com/wp-content/uploads/2021/04/Aurelien-Geron-Hands-On-Machine-Learning-with-Scikit-Learn-Keras-and-Tensorflow_-Concepts-Tools-and-Techniques-to-Build-Intelligent-Systems-OReilly-Media-2019.pdf)
+-[Machine Learning Tutorials -GeeksforGeeks](https://www.geeksforgeeks.org/machine-learning/)
+
+## Deep Learning and Neural Networks
+- [Deep Learning basics with Python, TensorFlow and Keras -Sentdex](https://youtube.com/playlist?list=PLQVvvaa0QuDfhTox0AjmQ6tvTgMBZBEXN)
+- [Deep Learning Crash Course -FreeCodeCamp](https://youtu.be/VyWAvY2CF9c)
+- [Keras with TensorFlow Course - Python Deep Learning and Neural Networks for Beginners Tutorial -FreeCodeCamp](https://youtu.be/qFJeN9V1ZsI)
+- [Neural Networks from Scratch in Python -Sentdex](https://youtube.com/playlist?list=PLQVvvaa0QuDcjD5BAw2DxE6OF2tius3V3)
+- [Practicle Deep Learning by Fast.ai -FreeCodeCamp](https://youtu.be/0oyCUWLL_fU)
+- [Neural Network Full Course](https://youtu.be/ob1yS9g-Zcs)
+- [ Convolutional Neural Networks for Visual Recognition -Standford(Youtube)](https://youtube.com/playlist?list=PL3FW7Lu3i5JvHM8ljYj-zLfQRF3EO8sYv)
+- [Deep Learning -by Standford online](https://youtube.com/playlist?list=PLoROMvodv4rOABXSygHTsbvUz4G_YQhOb)
+- [Deep Learning with PyTorch (9-Day Mini-Course)](https://machinelearningmastery.com/deep-learning-with-pytorch-9-day-mini-course/)
+
+## Spacelization in Machine Learning
+
+### Natural Language Processing
+- [Natural Language Processing with TensorFlow2 -by Freecodecamp](https://youtu.be/B2q5cRJvqI8)
+- [Natural Language Processing with Spacy and Python -By FreeCodeCamp](https://youtu.be/dIUTsFT2MeQ)
+- [Natural Language Processing with Python -Sentdex](https://youtube.com/playlist?list=PLQVvvaa0QuDf2JswnfiGkliBInZnIC4HL)
+- [Distributed Representations of Words and Phrases and their Compositionality" by Tomas Mikolov- Research Paper](https://proceedings.neurips.cc/paper/2013/file/9aa42b31882ec039965f3c4923ce901b-Paper.pdf)
+- [Natural Language Processing with Python by Steven Bird, Ewan Klein, and Edward Loper -Book pdf](https://tjzhifei.github.io/resources/NLTK.pdf)
+
+### Computer Vision and Image Processing
```

Comment:
Other good resources that should be listed:

OpenCV: https://pyimagesearch.com/
Computer Vision (OpenCV) Information: https://pyimagesearch.com/2018/07/19/opencv-tutorial-a-guide-to-learn-opencv/